### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jackboberg**
```
› npm outdated
Package  Current  Wanted  Latest  Location
code       1.5.0   1.5.0   4.0.0  modulus
lab        6.2.0   6.2.0  11.1.0  modulus
```
Both of those updates require node >= 4. I think that change should be handled separately.

Closes #153

### CODE REVIEW

**Comment by jackboberg** [comment]
There may be an issue with the `progress@1.1.8` update, doing some more testing to confirm.

**Comment by jackboberg** [comment]
rebased with [this change](https://github.com/onmodulus/modulus-cli/pull/154/files#diff-e5001988c284bf49302c6986b4b88a3bR322) that seems to resolve issue with `progress` dependency update.

**Comment by musgravejw** [comment]
👍

File: lib/controllers/project.js [hunk 1]
```diff
@@ -320,12 +319,7 @@ Project.prototype._deploy = function (projectId, file, options, callback) {
     if (process.stdout.isTTY) { ubar.tick((progress * 100) - ubar.curr); }
   });
 
-  client.on('error', function(err) {
-    //closes the ubar readline interface
-    ubar.rl.close();
-
-    callback(err);
-  });
+  client.on('error', callback);
```
Comment (theworkflow) [inline lib/controllers/project.js]: I've tried testing, but haven't been able to see the `ubar` *NOT* close. Since I haven't run into any problems 👍

**Pull request merged**